### ISSUE TITLE
Unable to Execute a BEAMnrc simulation with 178 control points in DYNVMLC file

### ISSUE THREAD

**Issue by vahx129** [nrc-cnrc/EGSnrc#1167]
**Describe the bug**
I'm trying to simulate a VMAT full arc in BEAMnrc on its own (not worried about gantry motion, just dynamic MLCs/jaws) but am getting "The job has failed." in BEAMnrc GUI or Seg faults when running in the command line. I find that when I truncate the number of dynamic MLC and jaw positions from 178 to 100, the simulation appears to run without issue. 

**To Reproduce**
A precise description of the steps to reproduce the behavior:

Generally: 

Run a BEAMnrc simulation with syncjaws and dynvmlc components that each point to a text file containing 178 control points worth of data. 

My specific situation:

Link to my files: https://1drv.ms/f/s!ArqLIYhAstNSvv9VK1bPyI5ivVwC0Q?e=1DX5Cp

Using TrueBeam BEAMnrc module and TJM pegs file (included in cloud link), run the following inputs:

1. Run BEAMnrc input SBRTProstate_001.egsinp while having syncjaws point to SBRTProstate_001_JAW.txt and dynvmlc point to SBRTProstate_001.txt. Simulation should fail as each have 178 control points in the txt file. 
2. Run the same BEAMnrc input SBRTProstate_001.egsinp while having syncjaws point to SBRTProstate_001_JAW_Trunc.txt and dynvmlc point to SBRTProstate_001_Trunc.txt. The simulation should now work since there are only 100 control points in the files. 

**Expected behavior**
Though I do have a MATLAB script writing the jaw/MLC dynamic files for me based on a DICOM-RT plan input, I *believe* that they are formatted correctly, so I would expect the BEAMnrc simulation to work even with 178 control points. 

**Screenshots**

**Operating system**

- [x ] Linux
- [ ] Windows
- [ ] macOS
- [ ] Other: 

**EGSnrc version**
[2023a release]

**Additional context**
I feel like I've run these kinds of simulations in the past without issue but it's been a little while and I may be rusty about things. I'm trying to simulate a TrueBeam linac with MLC120. I'm using a 6x above jaws phase space provided by Varian and have modeled the Y jaws, X jaws, and MLC myself. I use a MATLAB script to convert DICOM-RT plan files for 6x TrueBeam IMRT treatments into BEAMnrc compatible formats. Ultimately I want this BEAMnrc file to be a source for a dynamic simulation in DOSXYZnrc but I feel like it should run on its own first before going any further. 

![image](https://github.com/user-attachments/assets/c6dfc342-8971-43a8-82fb-ea5de483695c)
![image](https://github.com/user-attachments/assets/a2db8567-4b83-4915-a9cd-3bff57b642cd)

**Comment by vahx129** [comment]
I have actually discovered a couple of things I may have been doing wrong so I may close this one shortly and wouldn't waste any resources on it just yet. 

My apologies for rushing to the issues forum with this one! I was so confident in how my other accelerator model was working that some of the nuances in switching over to a new one (using the TrueBeam phase space instead of having the whole thing modeled) caught me off guard. 

I think I just about have everything working the way I want other than getting the following message multiple times before the simulation completed: 

 ***WARNING in +ve bar of SYNCJAWS 
 Particle is well above or below jaw, yet region number  
 indicates the particle is still in the jaw. 
 The particle is being discarded but this is a bug. 
 Please contact the NRC.
